module1/src/test/java/net/broscorp/valueref/FooTest.java

```diff
@@ -0,0 +1,43 @@
+package net.broscorp.valueref;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class FooTest {
+
+  // Java is always pass-by-value
+
+  Foo object = new Foo();
+  int number = 1;
+  List<String> listOfStrings = new ArrayList<>();
```

Comment:
Just a hint - better to init something in tests in a method annotated with @Before annotation. It's a more clear way